1-exercises/H-array-methods-2/exercise2.js

```diff
@@ -1,21 +1,27 @@
 /*
   Array methods - .join()
   -------------------------
-  Complete the capitalise function 
+  Complete the capitalize function 
```

Comment:
This is a very minor comment, but there is nothing wrong with the spelling of capitalise. Capitalise is british english while capitalize is american english.